src/browser-polyfill.js

```diff
@@ -13,7 +13,7 @@ if (typeof browser === "undefined") {
   // never actually need to be called, this allows the polyfill to be included
   // in Firefox nearly for free.
   const wrapAPIs = () => {
-    const apiMetadata = {/* include("api-metadata.json") */};
+    const apiMetadata = require("../api-metadata.json");
```

Comment:
But the `require` will not work correctly if the source version is loaded accidentally instead of the build version as a plain tag script added to an extension page, or as an AMD module.

We can prevent the "CommonJS" loader (and bundler) to pick the wrong file by changing the "main" property in the package.json to the dist file.

Let's wait a feedback from @kmaglione on this (but I vote for switching back to the `{/* include("...") */}` ;-)